src/main/scala/mesosphere/marathon/Features.scala

```diff
@@ -17,12 +17,16 @@ object Features {
   //enable GPUs
   lazy val GPU_RESOURCES = "gpu_resources"
 
+  //enable Mesos HealthChecks
+  lazy val MESOS_HEALTHCHECKS = "mesos_healthchecks"
+
   lazy val availableFeatures = Map(
     VIPS -> "Enable networking VIPs UI",
     TASK_KILLING -> "Enable the optional TASK_KILLING state, available in Mesos 0.28 and later",
     EXTERNAL_VOLUMES -> "Enable external volumes support in Marathon",
     SECRETS -> "Enable support for secrets in Marathon (experimental)",
-    GPU_RESOURCES -> "Enable support for GPU in Marathon (experimental)"
+    GPU_RESOURCES -> "Enable support for GPU in Marathon (experimental)",
+    MESOS_HEALTHCHECKS -> "Enable support for Mesos Native Healthchecks in Marathon (require Mesos version >= 1.1.0)"
```

Comment:
I think mesos health checks were introduced in `1.2.0` according to https://mesosphere.com/blog/2017/01/05/introducing-mesos-native-health-checks-apache-mesos-part-1/ ?